unix/mclock.ml

```diff
@@ -15,7 +15,7 @@
  *)
 type t = unit
 type 'a io = 'a Lwt.t
-type error = unit
+type _error = unit
```

Comment:
I prefer to put `[@@@warning "-34"]` or it's possible to just delete it?
NOTE: `type[@warning "-34"] error` does not work ... I will make an issue about that in OCaml.

Reply:
removed it in bbaaa918277463cfe366d4b811f73456a7a853e4 indeed :)